device/rocev2/rocev2.yaml

```diff
@@ -0,0 +1,23 @@
+components:
+  schemas:
+    Device.Rocev2Peer:
+      description: >-
+        Configuration for one or more IPv4 or IPv6 RoCEv2 Peers.
+      type: object
+      properties:
+        ipv4_interfaces:
+          description: >-
+            This contains an array of references to IPv4 interfaces,  each of which
+            will have list of peers to different destinations.   
```

Comment:
Replace By : This contains an array of references to IPv4 interfaces, each having a list of peers to various destinations.